docs/vectorizer-api-reference.md

```diff
@@ -602,7 +602,7 @@ HNSW is suitable for in-memory datasets and scenarios where query speed is cruci
 ```sql
   SELECT ai.create_vectorizer(
       'blog_posts'::regclass,
-      indexing => ai.indexing_hnsw(min_rows => 50000, opclass => 'vector_l2_ops'),
+      indexing => ai.indexing_hnsw(min_rows => 50000, opclass => 'vector_l1_ops'),
```

Comment:
@jgpruitt do you remember if there was a specific reason that we did not have `vector_l2_ops` in this list: https://github.com/timescale/pgai/blob/main/projects/extension/sql/ai--0.6.0.sql#L2112?

Reply:
We had a slack discussion on this before (I just forgot to push the docs change afterwards and remembered now):
> I believe the last three are relatively recent additions to pgvector and probably didn’t exist when our code was first written